TotalSegmentator/TotalSegmentator.py

```diff
@@ -803,7 +792,7 @@ def process(self, inputVolume, outputSegmentation, fast=True, task=None, subset=
         # Get TotalSegmentator launcher command
         # TotalSegmentator (.py file, without extension) is installed in Python Scripts folder
         import sysconfig
-        totalSegmentatorPath = os.path.join(sysconfig.get_path('scripts'), "TotalSegmentator")
+        totalSegmentatorPath = os.path.join(sysconfig.get_path('scripts'), "TotalSegmentator.exe")
```

Comment:
You don't need to use this launcher. It is just a shortcut for `python.exe -m TotalSegmentator`. Use the Python executable instead (see for example how `pip.exe` is launched by `pip_install` function).

Reply:
Please explain.